include/net/downloader.h

```diff
@@ -55,6 +55,7 @@ enum downloader_evt_id {
 	 * - -EINVAL: Invalid configuration.
 	 * - -EAFNOSUPPORT: Unsupported address family (IPv4/IPv6).
 	 * - -EHOSTUNREACH: Failed to resolve the target address.
+	 * - -EMSGSIZE: TLS packet is larger than the nRF91 modem can handle.
```

Comment:
For consistency
```suggestion
	 * - -EMSGSIZE: TLS packet is larger than the nRF91 Series modem can handle.
```
@divipillai